xilem/src/view/prose.rs

```diff
@@ -13,15 +13,26 @@ pub fn prose(content: impl Into<ArcStr>) -> Prose {
         text_brush: Color::WHITE.into(),
         alignment: TextAlignment::default(),
         text_size: masonry::theme::TEXT_SIZE_NORMAL as f32,
+        line_break_mode: LineBreaking::WordWrap,
     }
 }
 
+/// A version of [`prose`] suitable for including in the same line
+/// as other content.
+///
+/// Note that setting [`alignment`](Prose::alignment) on the result
+/// will be meaningless.
+pub fn inline_prose(content: impl Into<ArcStr>) -> Prose {
```

Comment:
Maybe `span` instead as I think it's shorter and similar to the web equivalent?

But `inline_prose` on the other hand is descriptive.